.eslintrc.js

```diff
@@ -25,9 +25,6 @@ module.exports = {
 	env: {
 		'jest/globals': true,
 	},
-	globals: {
-		wpApiSettings: true,
```

Comment:
Notice that I removed the global from the config and the test is still passing. Any idea why?

Reply:
I think it's `eslint-config-wordpress`:

https://github.com/WordPress-Coding-Standards/eslint-config-wordpress/blob/73a8d0a67ef714159b2df2aa16ce4d9180162761/index.js#L13

The `false` value is a bit misleading. It means that the variable _is_ considered a global, but it's not allowed to be replaced:

https://eslint.org/docs/user-guide/configuring#specifying-globals

Reply:
The behavior of `false` is a bit frustrating as well because it makes it unclear how a project like ours could override the default config to disallow the global. Maybe worth discussing in tomorrow's JS chat?

Reply:
We already said we'd provide a new eslint package in Gutenberg with the updated guidelines discussed in previous meetings. We can remove these globals entirely from this new package.

We can discuss removing the globals altogether (as part of our new JS coding standards for ESnext)

Reply:
>We already said we'd provide a new eslint package in Gutenberg with the updated guidelines discussed in previous meetings.

It's still an open question as to whether `wp` would be a global we want to include in the default configuration, assuming a plugin might expect to use those globals rather than import from npm packages.

Reply:
That's true, maybe we should keep it and just override it in Gutenberg (if possible).